model-client/src/commonMain/kotlin/org/modelix/model/client2/ReplicatedModel.kt

```diff
@@ -23,29 +23,47 @@ import org.modelix.model.operations.OTBranch
 
 /**
  * Dispose should be called on this, as otherwise a regular polling will go on.
+ *
+ * @property providedScope the CoroutineScope to use for the suspendable tasks
+ * @property lastKnownRemoteVersion the last version on the server from which we want to start the synchronization
+ * @property scope the CoroutineScope to use for the suspendable tasks. Uses providedScope or defaults to the default dispatchers
+ * @property state the current state of the ReplicatedModel
+ * @property localModel thread-safe maintenance of the local version
+ * @property remoteVersion to send and receive versions from the server
+ * @property pollingJob the coroutine job that periodically checks if there is a new remote version and fetches its content
```

Comment:
> @ property name
> 
> Documents the property of a class which has the specified name. This tag can be used for documenting properties declared in the primary constructor, where putting a doc comment directly before the property definition would be awkward.

https://kotlinlang.org/docs/kotlin-doc.html#property-name

Looks like properties declared in the class body should be documented directly.

Reply:
in [481f83d](https://github.com/modelix/modelix.core/pull/725/commits/481f83da1fef69c3b946773d198703fcae70ce8e), I documented the public API in the end.